cypress/e2e/with_mock_data/items.cy.ts

```diff
@@ -103,14 +103,21 @@ describe('Items', () => {
   it('adds an item with only mandatory fields', () => {
     cy.findByRole('button', { name: 'Add Item' }).click();
 
-    cy.findByLabelText('Usage status *').click();
-    cy.findByText('New').click();
+    cy.findByRole('progressbar').should('not.exist');
+
+    cy.findAllByText('Storage').first().click();
+
+    cy.findByText('No systems found').should('exist');
+
+    cy.findAllByText('Storage').should('have.length', 1);
+
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(100);
```

Comment:
Looks like this was needed in these tests because at the point it is trying to click next afterwards, the rules request hasn't finished so it errors as it triggers `!isDstSystemTypeSameAsSrcSystemType && (!SelectedRule || SelectedRule.length === 0)`, and even waiting for the request to finish wont necessarily be enough if it hasn't re-rendered the dialog yet.

A user would have to be very fast/the backend would have to get stuck handling a request for this to break for a user so I am not too concerned. I think the only long term solution I can think of now is having a loading spinner while it is still loading the relevant data for the form, so the tests can then wait for something to be displayed to know everything is ready.